tests/raftstore/transport_simulate.rs

```diff
@@ -125,7 +121,7 @@ impl<T: Transport> Transport for SimulateTransport<T> {
         }
 
         for strategy in self.filters.iter().rev() {
-            res = strategy.wl().after(res);
+            res = strategy.after(discard, res);
```

Comment:
I think @tiancaiamao want to keep before/after matching. 
E,g, two strategies, 1 and 2, in before, 1 return false, 2 return true, so in after, 1 should use false not true too.